testing/py3-requests-mock/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-requests-mock"
+_pyname="requests-mock"
+pkgver="1.6.0"
+pkgrel=0
+pkgdesc="Mock out responses from the requests package"
+url="https://pypi.org/project/requests-mock/"
+arch="noarch"
+license="Apache-2.0"
+depends="python3 py3-requests py3-six"
+makedepends="py3-pbr py3-mock py3-purl py3-testrepository"
```

Comment:
I think these should be in `checkdepends` https://github.com/jamielennox/requests-mock/blob/master/test-requirements.txt

Reply:
Fixed